release.toml

```diff
@@ -0,0 +1,13 @@
+owners = ["poljar", "dkasak"]
+
+pre-release-commit-message = "chore: Release {{crate_name}} version {{version}}"
+pre-release-replacements = [
+    {file="README.md", search="vodozemac = .*", replace="{{crate_name}} = \"{{version}}\""},
+]
+pre-release-hook = ["git", "cliff", "--prepend", "CHANGELOG.md", "--unreleased", "--tag", "{{version}}" ]
```

Comment:
Since this uses `git cliff` under the hood, could we mention it in the `RELEASING.md` file that it's a prerequisite to start a release?

Reply:
Good point!